widgets/blueprints/src/BlueprintUploadActionsMenu.tsx

```diff
@@ -9,7 +9,7 @@ const { TerraformModal } = Stage.Common;
 const { drilldownPage } = Stage.Common.Consts;
 const { UploadModal: UploadBlueprintModal } = Stage.Common.Blueprints;
 
-const t = Stage.Utils.getT('widgets.common.blueprintUpload.actionsMenu');
+const translateBlueprintUploadActionsMenu = Stage.Utils.getT('widgets.common.blueprintUpload.actionsMenu');
```

Comment:
Optional:

Not a must, but here we don't need to be unique, so simple:
```suggestion
const translate = Stage.Utils.getT('widgets.common.blueprintUpload.actionsMenu');
```
will also be OK.

Reply:
https://github.com/cloudify-cosmo/cloudify-stage/pull/2413/commits/fa9c4814d373d23cc0625efdbc52cdd1162bdc67 done ✔️